modern/src/SocketController.js

```diff
@@ -83,7 +77,27 @@ const SocketController = () => {
     return null;
   }, [authenticated]);
 
-  return null;
+  useEffect(() => {
+    events.forEach((event) => {
+      setNotification({
+        message: `${devices[event.deviceId]?.name}: ${t(prefixString('event', `${event.type}`))}`,
+        show: true,
+      });
+      setTimeout(() => setNotification({ message: '', show: false }), 6000);
+    });
+  }, [events]);
+
+  return (
+    <Snackbar
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'right',
+      }}
+      open={notification.show}
+      autoHideDuration={5000}
```

Comment:
Why do we have both `autoHideDuration` and `setTimeout`? Can you please explain how it works.